portals/application/modules/userweb/controllers/AuthController.php

```diff
@@ -55,11 +69,24 @@ public function postAction()
             }
         }
 
+        if ($gsQRCode) {
+            $extension = $user->getExtension();
+
+            if (!empty($extension)) {
+                $extensionNumber = $extension->getNumber();
+                $result['extensionNumber'] = $extensionNumber;
+            }
+        }
+
         $result['success'] = true;
         $result['message'] = 'Login Correcto';
         $result['userName'] = $user->getFullName();
         $result['companyName'] = $user->getCompany()->getName();
 
+        if ($gsQRCode) {
+            $result['companyDomain'] = $user->getCompany()->getDomain();
```

Comment:
I would combine those three conditions "if ($jsQRCode)" into a single one. What do you think about that?